x/leverage/keeper/keeper.go

```diff
@@ -604,3 +606,13 @@ func (k Keeper) iterate(ctx sdk.Context, prefix []byte, cb func(key, val []byte)
 
 	return nil
 }
+
+// WeightedOperations returns the all the leverage module operations with their respective weights.
```

Comment:
This can be moved to `app.go` with the signature

`func (am AppModule) WeightedOperations(simState module.SimulationState) []simtypes.WeightedOperation`

similar to how it's done [here](https://github.com/cosmos/cosmos-sdk/blob/master/x/distribution/module.go#L203), so keeper doesn't need to import simulation stuff

Reply:
I will do that, this way we need to add the `bankKeeper` to the  `AppModule` structure as well

Reply:
done at https://github.com/umee-network/umee/pull/401/commits/20cd42a30a344993c368a6c33163cc1a66b5d7b4